includes/ting_search_autocomplete.admin.inc

```diff
@@ -0,0 +1,210 @@
+<?php
+
+/**
+ * @file
+ * Admin settings for bibliotek.dk autocomplete.
+ */
+
+function ting_search_autocomplete_admin() {
+
+  $base_settings['index'] = NULL;
+  $base_settings['facetIndex'] = NULL;
+  $base_settings['filterQuery'] = NULL;
+  $base_settings['sort'] = 'count';
+
+  $base_settings['agency']  = variable_get('ting_agency', NULL);
+  $base_settings['profile'] = variable_get('ting_search_profile', NULL);
+  $base_settings['maxSuggestions'] = 10;
+  $base_settings['maxTime'] = 2000;
+
+  $base_settings['highlight'] = NULL;
+  $base_settings['highlightPre'] = NULL;
+  $base_settings['highlightPost'] = NULL;
+  $base_settings['minimumString'] = 3;
+
+  $defaults = variable_get('ting_search_autocomplete_settings', array());
+
+  foreach ($base_settings as $key => $base_setting) {
+    $default_settings[$key] = (!empty($defaults[$key])) ? $defaults[$key] : $base_setting;
+  }
+
+  // Attach the CSS to the form
+  $path = drupal_get_path('module', 'ting_search_autocomplete');
+  $form['#attached'] = array(
+    'css' => array(
+      'type' => 'file',
+      'data' => $path . '/css/ting_autocomplete_admin_form.css',
+    ),
+  );
+
+  $form['ting_search_autocomplete_webservice'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Autocomplete URL', array(), array('context' => 'ting_search_autocomplete')),
+    '#tree' => FALSE,
+    '#collapsible' => TRUE,
+  ); 
+
+  $form['ting_search_autocomplete_webservice']['ting_suggestion_url'] = array(
```

Comment:
See comment regarding [naming of variables](https://github.com/ding2/ting/pull/30#discussion_r28693954).